python/ntp/commands/run/ntuple/__init__.py

```diff
@@ -121,7 +121,6 @@ def __run(self, campaign, dataset):
             return False
 
     def __add_output_file(self, dataset, campaign, suffix):
-
         if self.__variables['output_file'] is not None:
```

Comment:
It would be better to change 
```python
if self.__variables['output_file'] is not None:
# to
if self.__variables['output_file']:
```
but your change does the job. The only problem would be if `output_file` is ever set to `True`